sqlsynthgen/main.py

```diff
@@ -167,7 +168,9 @@ def make_stats(
         typer.echo("Missing source database connection details.", err=True)
         sys.exit(1)
 
-    src_stats = make_src_stats(src_dsn, config)
+    src_stats = asyncio.get_event_loop().run_until_complete(
```

Comment:
If this change was purely for SSG then it sounds like `.run()` would be simplest but I guess we're calling `make_stats` from TAPAS so there might already be a loop running?

```suggestion
    src_stats = asyncio.get_event_loop().run_until_complete(
```

Reply:
Yeaaah, I think that was my logic here. I'm not super sure of my understanding of how things like multiple event loops work though.